samples/drivers/gpio/src/main.c

```diff
@@ -163,9 +184,17 @@ void main(void)
 	}
 
 	/* Setup GPIO input, and triggers on rising edge. */
+#ifdef CONFIG_SOC_CC2650
+	ret = gpio_pin_configure(gpio_dev, GPIO_INT_PIN,
+				 (GPIO_DIR_IN | GPIO_INT |
+				  GPIO_INT_EDGE | GPIO_INT_ACTIVE_HIGH |
+				  GPIO_INT_DEBOUNCE | GPIO_INT_PIN_EXTRAFLAGS));
```

Comment:
For the Sensortag, I needed an extra GPIO_PUD_PULL_UP flag.
As this is probably not the only board needing to pass some extra flags, I added a little bit of flexibility, so other boards in the future can define (or not) more flags as needed while keeping code readable. 
So yes, for now it's only used for the CC2650 but it may prove useful as the test code becomes used for more boards.